src/de/cuuky/varo/game/world/setup/AutoSetup.java

```diff
@@ -128,4 +130,40 @@ private void setupSpawns(Location middle) {
             new AutoStart(start);
         }
     }
-}
\ No newline at end of file
+
+    private Location getLobbyLocation(World world, long x, long z) {
+        return new Location(world, x, getLobbyHeight(world, x, z), z);
+    }
+
+    private int getLobbyHeight(World world, long x, long z) {
+        int heightSetting = ConfigSetting.AUTOSETUP_LOBBY_SNAP_OFFSET.getValueAsInt();
+        LobbySnap snapSetting = LobbySnap.valueOf(ConfigSetting.AUTOSETUP_LOBBY_SNAP.getValueAsString());
+
+        switch (snapSetting) {
+            case GROUND:
+                return getGroundHeight(world, x, z) + heightSetting;
+            case ABSOLUTE:
+                return heightSetting;
+            default:
+            case MAX_HEIGHT:
```

Comment:
```suggestion
            case MAX_HEIGHT:
            default:
```
This just looks confusing

Reply:
I have generally thought about the default case and came to the conclusion that it should generally throw a `NotImplementedException` and not fall back to a default case. This way a misimplementation can be noticed quickly.